src/modules/hr-recruiter/hr-recruiter.service.ts

```diff
@@ -20,4 +29,30 @@ export class HrRecruiterService {
   async findOne(id: string) {
     return await Recruiter.findOneByOrFail({ id });
   }
+
+  async reserveStudent(studentEmail: string, recruiterUserId: string) {
```

Comment:
tu myślę, że ta metoda powinna też od razu rekruterowi przypisywać tego studenta do tej tablicy rezerwowanych studentów (property reservedStudents).
Czy to będzie później lub gdzie indziej robione?